### PR TITLE
[Task-13][Frontend] As a user, I can upload CSV file

### LINKED ISSUE #13: [Task-13][Frontend] As a user, I can upload CSV file
## Why

User friendly uplaod form
 
## Acceptance Criteria

- [ ] Upload form
- [ ] Upload button

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-61408341

## What happened 👀

Upload form for CSV keywords

## Insight 📝

- [x] Frontend of upload form
- [x] Frontend of CSV file table
 
## Proof Of Work 📹

- Upload form
image

- Table
image

### CODE REVIEW

File: assets/css/_variables.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 $font-size: (
   sm: .875rem,   // 14px
   md: 1rem,      // 16px
+  xxxl: 1.5rem,  // 24px
```
Comment (hanam1ni) [inline assets/css/_variables.scss]: How about using `xl` instead?
```suggestion
  xl: 1.5rem,  // 24px
```

File: lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex [hunk 2]
```diff
@@ -1,28 +1,32 @@
-<h1>Listing Keyword files</h1>
+<div class="page-header">
```
Comment (hanam1ni) [inline lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex:1]: Let's fix the spacing in this file. It looks a bit wider.

Comment (hanam1ni) [reply lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex:1]: Also, fix in other files too.

Comment (rafayet-monon) [reply lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex:1]: Fixed issues in [54c58ea](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/54c58ea81b3b8862d92e925946d9dc725a1c90ff)

File: lib/elixir_search_extractor/file_upload.ex [hunk 3]
```diff
@@ -3,10 +3,13 @@ defmodule ElixirSearchExtractor.FileUpload do
   alias ElixirSearchExtractor.FileUpload.CsvUploader
   alias ElixirSearchExtractor.FileUpload.CsvValidator
   alias ElixirSearchExtractor.FileUpload.KeywordFile
```
Comment (rosle) [inline lib/elixir_search_extractor/file_upload.ex]:
```suggestion
  alias ElixirSearchExtractor.FileUpload.{CsvUploader, CsvValidator, KeywordFile}
```

File: test/elixir_search_extractor/file_upload_test.exs [hunk 4]
```diff
@@ -6,11 +6,16 @@ defmodule ElixirSearchExtractor.FileUploadTest do
   alias ElixirSearchExtractor.FileUpload
   alias ElixirSearchExtractor.FileUpload.KeywordFile
 
-  describe "list_keyword_files/0" do
-    test "returns all keyword_files" do
-      keyword_file = insert(:keyword_file)
+  describe "paginated_user_keyword_files/2" do
+    test "returns only user's keyword_files" do
+      user = user_fixture()
+      user_file = insert(:keyword_file, user: user)
```
Comment (rosle) [inline test/elixir_search_extractor/file_upload_test.exs:12]: Should we add another user keyword file to make sure it is not included in the result?

File: assets/css/components/_page-header.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.page-header {
+  display: flex;
+  justify-content: space-between;
+  height: rem(50px);
+  margin-bottom: rem(15px);
+
+  &__title {
+    font-size: map-get($font-size, 'xxxl');
```
Comment (rosle) [inline assets/css/components/_page-header.scss]: I saw only `sm`, `md` and `xl` in the variables file. Is this supposed to be `xl`? 🤔

File: test/elixir_search_extractor/file_upload_test.exs (hunk 4 above)
Comment (rafayet-monon) [reply test/elixir_search_extractor/file_upload_test.exs:12]: @rosle added in [14aebd7](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/14aebd7020bfc582d397d624e5346dc1ea563b8a)

File: assets/css/components/_page-header.scss (hunk 5 above)
Comment (rafayet-monon) [reply assets/css/components/_page-header.scss]: @rosle Forgot to fix in the rebase 🙈  fixed in [2a5b032](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/2a5b032c7a09dbbe2054cf271cfb587b79144810)

**Review by rosle — APPROVED** [review]

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**